client/components/ProductOverview/ProductOverview.jsx

```diff
@@ -1,17 +1,38 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
+import axios from 'axios';
 
-import mockProductStyles from '../../../MockData/ProductStyles';
 import ImageGallery from './ImageGallery';
 
-import '../../css/ProductOverviewCSS.css';
+import '../../css/ProductOverviewCSS.scss';
```

Comment:
Just a naming thing, but `ProductOverviewCSS` seems a little redundant considering it's in a `css` directory and it's got a `.scss`  file extension. Also, it's best practice to name your css files after the component that imports them. Here's an example from the docs https://reactjs.org/docs/faq-structure.html

Reply:
i took out the CSS in the file name. I will start creating separate scss files from now on for each of my components.